source/utilities.cc

```diff
@@ -212,6 +212,60 @@ namespace WorldBuilder
     }
 
 
+    std::vector<Point<2> >
+    get_bounding_box (const Point<2> &point,
+                      const std::vector<Point<2> > &point_list,
+                      std::size_t original_number_of_coordinates,
+                      const double buffer_around_fault)
+    {
+      std::vector<double> x_list(original_number_of_coordinates,0.0);
+      std::vector<double> y_list(original_number_of_coordinates,0.0);
+
+      for (size_t j=0; j<original_number_of_coordinates; ++j)
+        {
+          x_list[j] = point_list[j][0];
+          y_list[j] = point_list[j][1];
+        }
+
+      // Corner coordinates of the bounding polygon
+      std::vector<Point<2> > bounding_box (4, point);
+
+      // The bounding box currently is hard-coded to include buffer zone based on the spherical geometry.
+      const double bounding_buffer_zone = 2.* (buffer_around_fault) / (const_pi * 2. * 6371000.) ;
```

Comment:
There is no reason to compute this in this function since you already have a parameter for `buffer_around_fault`. I also think this can be more easily generalized for all spherical geometries since the calling function has access to both the `depth` and the `radius from the center`. Adding both will give you the radius of the sphere/earth.

Could you also add a bit of an explanation of what this is calculating?